### ISSUE TITLE
Unexpected warning message during test

### ISSUE THREAD

**Issue by JayKan** [aria-51/aria-lights-out#12]
@coleturner,

So I was trying to get our unit testing working last night. After adjusting the current `.babelrc` with `react-emotion` plugin, all unit tests started to pass & work, __BUT__ I was getting some strange warning in regards to __multiple renderers__:
 
`Detected multiple renderers concurrently rendering the same context provider. This is currently unsupported`

To reproduce the same error message, add the following line in your `.babelrc` file: 

```javascript
"plugins": [
	["emotion", { "sourceMap": true, "autoLabel": true }]
]
```
and then run `npm t` or `yarn test`, you should see the warning message. 

After some digging, I was able to get rid of the warning message by removing `ReactDOM.render()` line. There seems to be a strange bug going on with `ReactDOM.render()` working in conjunction with `React.createContext()` in the same test file. I figure you might be interested in understanding this as well so [here you go](https://github.com/facebook/react/issues/13177) 🎉

**Comment by coleturner** [comment]
Yeah I noticed this when I was setting up the initial tests. The problem is that there is a primary (ReactDOM) and secondary (react-test-renderer) renderers which I've found an open bug for:
https://github.com/facebook/react/issues/13150

We can fix it by getting rid of ReactDOM since we don't need it (came with CRA IIRC). Let's close the issue for them since they've got it resolved in a related PR.

Fixed by:
https://github.com/aria-51/aria-lights-out/commit/48b659f38226aea55c7b46401b1ae90e06334baa

I've also added a `test` environment to `.babelrc` which fixes the emotion issue.
